dotnet/src/SemanticKernel.Abstractions/Functions/KernelArgumentsExtensions.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+namespace Microsoft.SemanticKernel;
+
+/// <summary>Provides extension methods for working with <see cref="KernelArguments"/> string values.</summary>
+public static class KernelArgumentsExtensions
```

Comment:
_If_ it was deemed important to support something here, my preference would be:
1. Not extension methods: if the extension methods class is in the same namespace and assembly as the class it's extending, the members could just be instance methods
2. The methods not be specific to `string` but instead be generic.